### ISSUE TITLE
Crash when closing window during repo check

### ISSUE THREAD

**Issue by TheReal1604** [borgbase/vorta#386]
**Describe the bug**
Program Crash

**To Reproduce**
OS: MacOS Catalina
1. Open Vorta
2. Start a borg repository check
3. Go to another window (safari, finder)
4. Open Vorta from the bar again
5. crash

**Desktop (please complete the following information):**
 - OS: Mac OS Catalina

**Comment by amarendra** [comment]
It's basically crashes when it's doing anything related to remote repo and you do anything with the open window or click on menu bar icon. If there's an email I could share Mac crash reports on that.

**Comment by m3nu** [comment]
Generally, Borg runs in a separate thread and you can keep using the main window as before.

Are you using the latest update? Feel free to send the log to **hello [a] borgbase.com**.

**Comment by m3nu** [comment]
The crash happens in Qt, rather than Python. Could be due to the translations `vorta.i18n - DEBUG - Loading translation failed for ['C']`.

You're also using `rsync.net`. Still necessary to set the remote borg binary with these guys?

```
Thread 0 Crashed:: Dispatch queue: com.apple.main-thread
0   libsystem_kernel.dylib        	0x00007fff689887fa __pthread_kill + 10
1   libsystem_pthread.dylib       	0x00007fff68a45bc1 pthread_kill + 432
2   libsystem_c.dylib             	0x00007fff6890fa1c abort + 120
3   QtCore                        	0x000000010a611de9 qt_message_fatal(QtMsgType, QMessageLogContext const&, QString const&) + 9
4   QtCore                        	0x000000010a613534 QMessageLogger::fatal(char const*, ...) const + 202
5   QtCore.abi3.so                	0x000000010a443cb2 pyqt5_err_print() + 802
6   QtCore.abi3.so                	0x000000010a449804 PyQtSlotProxy::unislot(void**) + 100
```

```
2020-03-17 10:25:44,874 - vorta.borg.borg_thread - INFO - Running command /usr/local/bin/borg prune --list --info --log-json --keep-hourly 2 --keep-daily 7 --keep-weekly 4 --keep-monthly 6 --keep-yearly 2 --prefix machine-machine- --keep-within 10H xxxx@xxxx.rsync.net:sink
2020-03-17 10:49:53,675 - vorta.i18n - DEBUG - Loading translation failed for ['C'].
2020-03-17 10:49:53,747 - apscheduler.scheduler - INFO - Scheduler started
2020-03-17 10:49:54,310 - vorta.borg.borg_thread - INFO - Running command /usr/local/bin/borg --version
```

**Comment by amarendra** [comment]
I saw it in their email so I am just using it. I have started using borg/rsync.net/vorga a day ago so I am not entirely sure. Since the app crashed I thought I'd share the logs. (Also, just realised Vorta might not be stable yet and I am using this to backup my personal data, I will try to configure bare-bone borg)

**Comment by m3nu** [comment]
I couldn't reproduce this even when using an invalid 'C' locale. Else we could just use English as fallback language. So not sure what causes this.

**Comment by amarendra** [comment]
> I couldn't reproduce this even when using an invalid 'C' locale. Else we could just use English as fallback language. So not sure what causes this.

Alright. My borg over bare bone script isn't doing much better either. I am getting disconnected from rsync.net continuously (talking to support over there). So next weekend I will again install vorta and see if it the crash occurs again and if it does I will share the logs. I will look at the dogs to enable full debug log (if there's a need). Please let me know if you'd want anything specific.

**Comment by m3nu** [comment]
If you have issues at the network layer, you may want to use a tool like iperf3 to debug that.

**Comment by amarendra** [comment]
> Are you using the latest update? Feel free to send the log to **hello [at] borgbase.com**.

Yes 0.6.26.

I have shared two crash logs along with the reproduction steps. 

1. Just the original steps in this issue. It's reproducible.
2. Another: select an archive → click delete → confirm → click to close button of Vorta window.
→ crashes

**Comment by deermo** [comment]
I actually encountered the same behaviour, and it seems to represent the same bug.
I use Vorta (currently, pip build, 0.6.26) and have observed it to crash when I manually want to check my repository. I have a local repository with around 40 archives, and  thus tried to check the respistory via command line, where all tests are passed.

When I manually check the repository from the Vorta GUI and close the Vorta window, Vorta crashes (as does python when I run Vorta from command line, I shared the macOS crash report for Python via e-mail).
However, the borg process however keeps in the background and the repository stays locked (I actually see two "borg-macosx64" processes). When writing this, the process actually is still running, but as I recall from last time, it did run for around 30 minutens and then unlocked the repo, which I interpret as borg finishing the validation.

The Vorta log shows nothing after:
```
2020-07-26 12:43:05,430 - vorta.borg.borg_thread - INFO - Running command /usr/local/bin/borg check --info --log-json user@domain:/path/to/repo
```
